source/code/source/playfab/PlayFabEventPipeline.cpp

```diff
@@ -225,6 +225,15 @@ namespace PlayFab
                 std::this_thread::sleep_for(std::chrono::milliseconds(this->settings->readBufferWaitTime));
             }
         }
+        // batch up any buffered events
+        while( PopBuffer() )
```

Comment:
Just ran this through a generate/build real quick. PopBuffer needs to be added to PlayFabEventPipeline.h as well.